mempool/src/feerate.rs

```diff
@@ -0,0 +1,54 @@
+use common::primitives::amount::Amount;
+
+lazy_static::lazy_static! {
+    pub(crate) static ref INCREMENTAL_RELAY_FEE_RATE: FeeRate = FeeRate::new(1000);
+    pub(crate) static ref INCREMENTAL_RELAY_THRESHOLD: FeeRate = FeeRate::new(500);
+}
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+pub(crate) struct FeeRate {
+    tokens_per_kb: u128,
+}
+
+impl FeeRate {
+    pub(crate) fn new(tokens_per_kb: u128) -> Self {
+        Self { tokens_per_kb }
+    }
+
+    pub(crate) fn of_tx(fee: Amount, tx_size: usize) -> Self {
+        let tokens_per_byte = Self::div_up(fee.into(), tx_size);
+        Self {
+            tokens_per_kb: tokens_per_byte * 1000,
+        }
+    }
+
+    pub(crate) fn compute_fee(&self, size: usize) -> Amount {
+        Amount::from(self.tokens_per_kb * u128::try_from(size).unwrap() / 1000)
```

Comment:
`unwrap` is forbidden in any context other than tests

Reply:
Would `expect` be OK here?

Reply:
Same discussion from before about static asserts. If you can make this work with static assert, then perfect, and unwrap would be acceptable. If not, then move this to its own line and use an expect.

Reply:
Moved to a separate line and used expect.